src/Plivo/Resources/Message/MessageInterface.php

```diff
@@ -86,15 +87,15 @@ protected function getList($optionalArgs = [])
             $this->uri,
             $optionalArgs
         );
-
         $messages = [];
-
         foreach ($response->getContent()['objects'] as $message) {
             $newMessage = new Message($this->client, $message, $this->pathParams['authId'], $this->uri);
-
             array_push($messages, $newMessage);
         }
-
+        // handle list empty case
+        if (empty($messages)){
+            return $response->getContent();
```

Comment:
This is a behavioural change. Why is this required?

Reply:
when we pass empty array,we are getting exception Fatal error: Uncaught TypeError: Argument 2 passed to Plivo\Resources\ResourceList::__construct() must be of the type array, null given, called in /root/plivo-php/src/Plivo/Resources/Message/MessageList.php on line 24 and defined in /root/plivo-php/src/Plivo/Resources/ResourceList.php:45
Stack trace:
#0 /root/plivo-php/src/Plivo/Resources/Message/MessageList.php(24): Plivo\Resources\ResourceList->__construct(Object(Plivo\MessageClient), NULL, Array)
#1 /root/plivo-php/src/Plivo/Resources/Message/MessageInterface.php(99): Plivo\Resources\Message\MessageList->__construct(Object(Plivo\MessageClient), NULL, Array)
#2 [internal function]: Plivo\Resources\Message\MessageInterface->getList(Array)
#3 /root/plivo-php/src/Plivo/Resources/ResourceInterface.php(65): call_user_func_array(Array, Array)
#4 /root/plivo-php/test.php(8): Plivo\Resources\ResourceInterface->__call('list', Array)

it's not handled previously.  Added to empty case check(bug fix).

When there is empty list, we are passing the empty response to customer